README.md

```diff
@@ -212,7 +212,7 @@ make test-python
 
 To develop Relay with an existing Sentry devserver, self-hosted Sentry
 installation, or Sentry SaaS, configure the upstream to the URL of the Sentry
-server in `.relay/config.yml`. For example, in local development set
+server in `./.relay/config.yml`. For example, in local development set
```

Comment:
Maybe it's better to be even more explicit, e.g.: 
```suggestion
server in `.relay/config.yml` in the project root directory. For example, in local development set
```

Reply:
Sounds good